lib/decimal.ml

```diff
@@ -490,6 +490,26 @@ let to_string ?(eng = false) ?(context = !Context.default) = function
     in
     Sign.to_string sign ^ intpart ^ fracpart ^ exp
 
+let to_decimal_string = function
```

Comment:
Would it be possible to fold this function directly into `to_string` and change  the latter's signature to e.g. `` val to_string : ?format:[`eng | `plain] -> ?context:Context.t -> string ``?

It looks like `to_decimal_string` is sharing a lot of the implementation code of `to_string`, so maybe let's cut out the duplication?

Regarding the signature change, I can bump up the version to `1.0.0` to indicate a backward-incompatible change. But I would wager that approximately no one is using the `?eng` argument right now anyway.

Reply:
Ah yeah that was how I initially wanted to implement it but thought I would preserve backwards compatibility. I think maybe making the options
```ocaml
type format = [
  `standard (* current format *)
  | `eng (* eng *)
  | `decimal (* or `plain -- *)
]
```

with `` `standard `` as the default seems like a good API.

Reply:
OK -- I've done this in the latest commit (0252f83) -- let me know if that looks better to you!

Reply:
OK -- I've done this in the latest commit (0252f83) -- let me know if that looks better to you!